### PR TITLE
added my comment to update caption.

### PR DESCRIPTION
**Pull request by pierremartindumont**
Please review my addition to the file! Thanks in advance! This resolve issue #4.

## To do 
- [ ] merge our branch
- [ ] delete the branch after we're done

### CODE REVIEW

File: _slides/12-pierremartindumont.md [hunk 1]
```diff
@@ -3,7 +3,8 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+This file was hidden on my file system, but finally got it!
+Well, my bad, it was under the _Slides folder.
```
Comment (KeithWinovich-eaton) [inline _slides/12-pierremartindumont.md:7]: Comment from KW

**Review by vmanuppelli — APPROVED** [review]
This looks really good!  APPROVED.

**Review by KeithWinovich-eaton — DISMISSED** [review]
Perhaps add more comments

File: _slides/12-pierremartindumont.md (hunk 1 above)
Comment (pierremartindumont) [reply _slides/12-pierremartindumont.md:7]: hey Keith, thanks for providing a 2nd opinion on my edits.

**Review by cody-carlson — APPROVED** [review]

**Comment by pierremartindumont** [comment]
Been reviewed by Vincent and Keith. Thanks guys!

**Pull request merged**